meson.build

```diff
@@ -0,0 +1,28 @@
+project('FabIO',
+  'c', 'cython',
+  license: 'MIT',
+  meson_version: '>= 0.60',
+  version: run_command('version.py',
+  			           check:true).stdout().strip(),
+)
+
+cc = meson.get_compiler('c')
+
+m_dep = cc.find_library('m', required : false)
+if m_dep.found()
+  add_project_link_arguments('-lm', language : 'c')
+endif
+
+# https://mesonbuild.com/Python-module.html
+py_mod = import('python')
+py = py_mod.find_installation()
+py_dep = py.dependency()
+
+py.install_sources(
+  ['version.py'],
+  #rename: ['_version.py'],
```

Comment:
You should be able to do this with 

```meson
version_py = configure_file(input: 'version.py', output: '_version.py', copy: true)
py.install_sources(
    version_py,
    pure: false,
    subdir: 'fabio',
)
```

However, you could also move this to fabio/meson.build, use `input: '../version.py'`, and simply add `version_py` to `python_sources`.